src/plugins/core/chart.ts

```diff
@@ -210,30 +233,32 @@ export class ChartPlugin extends CorePlugin<ChartState> implements ChartState {
   // Getters
   // ---------------------------------------------------------------------------
 
-  getChartDefinition(figureId: UID): ChartDefinition | undefined {
-    return this.chartFigures[figureId];
+  getChartDefinition(sheetId: UID, figureId: UID): ChartDefinition | undefined {
```

Comment:
It's an API change in a getter in stable, but honestly I do not see another solution :/

Reply:
the other solution consists in forcing the unicity of the figure (and not the combination of sheetid/figureid)  and store the sheetId in the figure definition ( go from  `{[sheetId]: Record<UID,figure>}` to  `Record<UID, {...figure, sheetId}>`but then we would need to migrate the data of the client, somehow convert the doubled ids, both in the data AND the revisions.  So unless we adapt the 'migrate' function to spit out the converted data as long as a table with the converted ids, I don't know how we could manage it